# POTD (Pick of the Day) - DOUBLE DOWN!!!
### Double Down - Nolan Arenado & Charlie Blackmon

### Random Forest:

In [1]:
import pandas as pd
import plotly.plotly as py
import plotly.figure_factory as ff
import plotly.graph_objs as go

rf = pd.DataFrame([['Charlie Blackmon', 0.305], 
                   ['Ian Desmond', 0.303], 
                   ['Pat Valaika', 0.296], 
                   ['Nolan Arenado', 0.291], 
                   ['Lorenzo Cain', 0.285]],
columns= ['Player', 'Probability']
)

print(rf)

             Player  Probability
0  Charlie Blackmon        0.305
1       Ian Desmond        0.303
2       Pat Valaika        0.296
3     Nolan Arenado        0.291
4      Lorenzo Cain        0.285


### Logistic Regression:

In [2]:
lr = pd.DataFrame([['Nolan Arenado', 0.352], 
                   ['Lorenzo Cain', 0.333], 
                   ['Freddie Freeman', 0.319], 
                   ['Kole Calhoun', 0.316], 
                   ['Charlie Blackmon', 0.316]],
columns= ['Player', 'Probability']
)

print(lr)

             Player  Probability
0     Nolan Arenado        0.352
1      Lorenzo Cain        0.333
2   Freddie Freeman        0.319
3      Kole Calhoun        0.316
4  Charlie Blackmon        0.316


#### Nolan Arenado

Arenado will face Hyun-Jin Ryu. He is 12/21 (.571) against him with a handedness advantage.

#### Charlie Blackmon

Blackmon is 8/27 (.296) against Ryu. This game will be in Coors too.